src/tiler/sql/user_progress.sql

```diff
@@ -0,0 +1,26 @@
+(SELECT
+  DISTINCT ON (block.id)
+  block.geom, block.id, turf.group_id,
+  CASE
+    WHEN (survey.user_id IS NOT DISTINCT FROM <%= user_id %>
```

Comment:
`IS NOT DISTINCT FROM` is essentially "null-safe" equality.  See the bottom of [this page](http://www.postgresql.org/docs/9.1/static/functions-comparison.html) for more info